ops/scripts/celo-update-op-geth.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+import subprocess
+import os
+import sys
+
+def main():
+	if len(sys.argv) < 2:
+		print('Please provide an op-geth commit or branch name')
+		sys.exit(1)
+
+	version = sys.argv[1]
+	for project in ('.',):
+		print(f'Updating {project}...')
```

Comment:
Looking forward to the output `Updating ....`.

Reply:
Hehehe!